docs/root/configuration/http/http_filters/stateful_session_filter.rst

```diff
@@ -70,9 +70,10 @@ So let's take this as an example.
       name: envoy.http.stateful_session.cookie
       typed_config:
         "@type": type.googleapis.com/envoy.extensions.http.stateful_session.cookie.v3.CookieBasedSessionState
-        name: global-session-cookie
-        path: /path
-        ttl: 120s
+        cookie:
```

Comment:
ideally these configs are in a yaml file (with full envoy config)

that way they can be validated

Reply:
i can follow up with this